### ISSUE TITLE
Documentation deprecated jade Support

### ISSUE THREAD

**Issue by M3ssman** [pugjs/pug#2897]
**Pug Version:** 2.0.0-rc.4

**Node Version:** v6.9.4

## Additional Comments
As the Migration Guide states, [https://pugjs.org/api/migration-v2.html](https://pugjs.org/api/migration-v2.html) 

> Although .jade is still supported, it is deprecated.

Anyway, I encountered a different Behaviour, meaning it was necessary to rename all *.jade to *.pug since it was looking only for Files with  *.pug Extension. Or did I miss something?

**Comment by droooney** [comment]
Probably it means that you can render or include `*.jade` files, but you really should rename all your `*.jade` files to `*.pug`. And could you post an example of "different behaviour" that you encountered?

**Comment by M3ssman** [comment]
Yes, you are right, 
that is at least what I expected. But, actually it did not work until I renamed all files from *.jade to *.pug. When I started my IDE (VS Code) in Debugging Mode it cried a thousand Bytes for missing *.pug Files, which simply not existed at this point.

Maybe it also depends on the setup for the rendering engine inside Express:
```
this.app.set('views', path.join(this.rootDir, 'views'));
this.app.set('view engine', 'pug');
```
I didnt investigate this any further, since renaming the files did the Job, too.  It could be a recent change in the pug API that's not quite documented yet (which is quite common).
